csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -6020,7 +6022,7 @@ A CSAF asset matching system satisfies the "CSAF asset matching system" conforma
     * when a new asset is inserted (for this asset)
     * when the Major version in a CSAF document with semantic versioning changes (for this CSAF document)
     > These also apply if more than one CSAF document or asset was added. To reduce the computational efforts the runs can be pooled into one run which fulfills all the tasks at once (batch mode).
-  > Manually and automatically triggered runs should not be pooled.
+  > Manually and automatically triggered runs SHOULD not be pooled.
```

Comment:
Ibid rephrase, lowercase, or augment text part to normative

Reply:
Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).